docs/source/concepts/dxf.rst

```diff
@@ -77,7 +77,7 @@ These are some application which do fir the criteria of a reliable CAD applicati
 
 Unfortunately, I cannot recommend any open source applications because everyone
 I know has serious shortcomings, at least as a DXF viewer, and I don't trust
-them as a DXF creator either. To be clear, even `ezdxf` (which is not a CAD
+them as a DXF creator either. To be clear, not even `ezdxf` (which is not a CAD
```

Comment:
It certainly seemed like this was missing from surrounding context - not at all deep enough in CAD & DXF to judge this otherwise 😄